lastpasslib/secrets.py

```diff
@@ -198,8 +219,7 @@ def shared_to_people(self):
                     'method': 'cr',
                     'token': self._lastpass.token}
             response = self._lastpass.session.post(url, data=data)
-            if not response.ok:
-                response.raise_for_status()
+            response.raise_for_status()
             sent = response.json().get('sent')
```

Comment:
Would we want to raise here or just log an error, since we return a boolean? I lean more towards the not raising to be honest...